CONTRIBUTING.md

```diff
@@ -13,6 +13,10 @@ This document describes how to contribute to Batarang
 
 ## Running the tests
 
+## Creating a PR
+
+Please follow the code & commit guidelines found in [Angular core's CONTRIBUTING.md](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
```

Comment:
is it possible to make this relative? since if for example you are editing contributing.md you'd like it to be relative to the fork/branch you're in